app/lib/send-email.js

```diff
@@ -46,27 +28,21 @@ const sendEmail = async (email, personalisation, reference, templateId, carbonEm
 
 const sendCarbonCopy = async (templateId, personalisation) => {
   if (config.notify.carbonCopyEmailAddress) {
-    await send(
+    await sendSFDEmail(
       templateId,
       config.notify.carbonCopyEmailAddress,
       personalisation
     )
   }
 }
 
+// As it's not obvious - this function send a message out to document generator for the application
+// It is NOT sending the email here.
 export const sendFarmerConfirmationEmail = async (emailParams) => {
```

Comment:
Maybe we could rename the function?

Reply:
I could do and did consider this, however the entire area probably needs renaming as actually nothing in here is doing any sending of emails, it is either requesting SFD messaging proxy to forward something on, or requesting document generator to forward something on. I'll have a further think this afternoon on how far to take that